packages/common/src/interaction.ts

```diff
@@ -61,12 +61,33 @@ export enum ExecutionRunStatus {
     failed = "failed",
 }
 
+/**
+ * @deprecated Use RunDataStorageLevel instead (since 0.39.1)
+ */
 export enum RestrictionLevel {
```

Comment:
It can be deleted no ?

Reply:
I will remove it in a separated PR to avoid breaking the build of Studio